cmake/libvpx.cmake

```diff
@@ -69,7 +105,13 @@ else()
         )
     endif()
 
-    set(ASM_SUFFIX ".asm.S")
+endif()
+
+if (ads2gas)
+    find_package(Perl)
+    if (NOT PERL_FOUND)
+        message(FATAL_ERROR "Unable to find perl, needed by ${ads2gas} on 32-bit ARM systems when NEON SIMD is enabled.")
```

Comment:
why not just `find_package(Perl REQUIRED)`?

Reply:
I guess my lack of experience with cmake is showing :smile:  Certainly, we can do that.  I guess one advantage the current approach has is that it explains _why_ perl is needed.

Reply:
i guess anyone who want to understand why it is needed can look into cmake files, cmake show the line what triggered the error AFAIR

Reply:
Changed to `find_package(Perl REQUIRED)`.